hack/make/ubuntu

```diff
@@ -150,6 +158,7 @@ EOF
 			--deb-recommends ca-certificates \
 			--deb-recommends git \
 			--deb-recommends xz-utils \
+			--deb-recommends apparmor \
```

Comment:
@tianon  you make the recommendation on #13156 that a Debian user using the "ubuntu" FPM packages might get AppArmor installed when they don't want it. I suppose that's true. However, they *should* want AppArmor or SELinux and I think recommending them is reasonable. If the argument is that the user might prefer SELinux and this package is not flexible enough... perhaps we should have an lxc-docker-lsm package that specifies the recommendation with provided-by lxc-docker-lsm-apparmor lxc-docker-lsm-selinux? Then, installs could happen with neither, or could satisfy the apt recommendation by either LSM module.

This seems complicated, but seems to be within the model of Debian packaging semantics.